src/MyEventEmitter.js

```diff
@@ -1,12 +1,79 @@
 'use strict';
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  on(eventName, listener) {
+    if (!this.events[eventName]) {
+      this.events[eventName] = [];
+    }
+    this.events[eventName].push(listener);
+  }
+
+  once(eventName, listener) {
+    const onceTimeListener = (...args) => {
+      listener(...args);
+
+      this.off(eventName, onceTimeListener);
+    };
+
+    this.on(eventName, onceTimeListener);
+  }
+
+  off(eventName, listener) {
+    if (!this.events[eventName]) {
+      return;
+    }
+
+    this.events[eventName].filter(listenerName => listenerName !== listener);
```

Comment:
The filter function returns a new array but it is not being assigned to anything. So the original array remains unchanged. You should assign the filtered array back to 'this.events[eventName]'.